File: js/calc.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question("Choose a number: ", (num1Input) => {
  const num1 = parseFloat(num1Input);

  rl.question("Choose another number: ", (num2Input) => {
    const num2 = parseFloat(num2Input);

    console.log("Pick an operation:");
    console.log("1. Addition");
    console.log("2. Subtraction");
    console.log("3. Multiplication");
    console.log("4. Division");

    rl.question("Pick an operation: ", (operationInput) => {
      const operation = parseInt(operationInput);
      let result;

      if (operation === 1) {
        result = num1 + num2;
      } else if (operation === 2) {
        result = num1 - num2;
      } else if (operation === 3) {
        result = num1 * num2;
      } else if (operation === 4) {
        result = num1 / num2;
      } else {
        console.log("Invalid Operation!!!");
        rl.close();
        return;
      }

      console.log("Result:", result);
      rl.close();
    });
  });
});

File: java/calc.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.print("Pick a number: ");
        Float num1 = Float.parseFloat(input.nextLine());

        System.out.print("Pick another number: ");
        Float num2 = Float.parseFloat(input.nextLine());

        System.out.println("Pick an operation: ");
        System.out.println("1. Addition");
        System.out.println("2. Subtraction");
        System.out.println("3. Multiplication");
        System.out.println("4. Division");       
        System.out.print("Choose one: "); 

        int operation = Integer.parseInt(input.nextLine());
        input.close();
        if (operation == 1) {
            Float result = num1 + num2;
            System.out.println(result);
        } else if (operation == 2) {
            Float result = num1 - num2;
            System.out.println(result);
        } else if (operation == 3) {
            Float result = num1 * num2;
            System.out.println(result);
        } else if (operation == 4) {
            Float result = num1 / num2;
            System.out.println(result);
        } else {
            System.out.println("Invaid operation!!!");
        }
    }
}